### PR TITLE
rename LineChart panel to TimeSeriesChart

### PR DESCRIPTION
**Pull request by sjcobb**
As [discussed in the Matrix chat](https://matrix.to/#/!XNmgYIAndZOkEvQAbb:matrix.org/$IVaC8SRjGxTE7tcwTe9uahcY1RZu7aoTu3pJH52F9UU?via=matrix.org), this PR renames our `kind: 'LineChart'` panel to `kind: 'TimeSeriesChart'`. This is a follow up to #573 which makes our names more explicit about the type of data they support. CUE schema is also updated and I did a find & replace in BE code (hopefully I didn't miss any)

### CODE REVIEW

File: docs/dashboard.md [hunk 1]
```diff
@@ -213,7 +213,7 @@ Example:
 
 ##### LineChart
```
Comment (Nexucis) [inline docs/dashboard.md:214]: `TimeseriesChart` ?

File: dev/data/dashboard.json [hunk 2]
```diff
@@ -39,7 +39,7 @@
       },
       "panels": {
         "upByLabel": {
-          "kind": "LineChart",
+          "kind": "TimeSeriesChart",
```
Comment (Nexucis) [inline dev/data/dashboard.json:42]: I would have written `Timeseries` I thought it was a single word, but I'm not the best in english here 😅

**Review by Nexucis — APPROVED** [review]
Thank you so much @sjcob you changed it everywhere including in the cue schema and in the go code !

**Review by LukeTillman — APPROVED** [review]
Oh man, you're making me look bad. I should've updated cue too. 😂

File: dev/data/dashboard.json (hunk 2 above)
Comment (sjcobb) [reply dev/data/dashboard.json:42]: Yeahh `TimeseriesChart` looks really weird to me and I kind of piggybacked off @LukeTillman's `TimeSeriesQuery` change too so those would be consistent

**Review by eunicorn — APPROVED** [review]

File: dev/data/dashboard.json (hunk 2 above)
Comment (sjcobb) [reply dev/data/dashboard.json:42]: Maybe worth another discussion, but I'm going to go ahead and merge this--if we decide to make the 'S' lowercase, I think that should be a follow-up PR where TimeSeriesQuery is also changed

**Pull request merged**